<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1311</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Table">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выбор операции &lt;br/&gt;по которой будет происходить расчёт&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <item>
       <property name="text">
        <string>A +B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>A - B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>A * B</string>
       </property>
      </item>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>39</y>
        <width>21</width>
        <height>541</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>100</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Удалит Полином A с заданным ключём из заданной таблицы.</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>289</y>
        <width>18</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;= &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ключ  A:</string>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>851</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>581</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ключ Полинома АB&lt;br/&gt;Он не должен повторяться с теми, &lt;br/&gt;что уже есть в выбранной таблице&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>851</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ключ Полинома B&lt;br/&gt;Он не должен повторяться с теми, &lt;br/&gt;что уже есть в выбранной таблице&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ключ  AB:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ключ Полинома А&lt;br/&gt;Он не должен повторяться с теми, &lt;br/&gt;что уже есть в выбранной таблице&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ключ  B:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>320</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Рассчитает по выбранному действию &lt;br/&gt;операции с двумя полиномами A и B&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Рассчитать</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>20</y>
        <width>401</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Hash Table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Sort Table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser_3"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Scan Table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser_4"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>60</y>
        <width>181</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Добавит полином &lt;br/&gt;в выбранную таблицу &lt;br/&gt;(справа)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Добавить в</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выбор в какую таблицу&lt;br/&gt;будет добавлен Полином.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Hash Table</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sort Table</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Scan Table</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>All Tables</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Полином_A:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>259</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Действия с полиномами</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>210</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Удалит Полином B с заданным ключём из заданной таблицы.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>289</y>
        <width>181</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Добавит полином &lt;br/&gt;в выбранную таблицу &lt;br/&gt;(справа)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Добавить в</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_4">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выбор в какую таблицу&lt;br/&gt;будет добавлен Полином.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Hash Table</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sort Table</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Scan Table</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>All Tables</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Полином_B:</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>851</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>280</y>
        <width>581</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>170</y>
        <width>181</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Добавит полином &lt;br/&gt;в выбранную таблицу &lt;br/&gt;(справа)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Добавить в</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выбор в какую таблицу&lt;br/&gt;будет добавлен Полином.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Hash Table</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sort Table</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Scan Table</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>All Tables</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>851</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>320</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Удалит Полином AB с заданным ключём из заданной таблицы.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>391</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Впишите свои полиномы в данные поля:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Log программы:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>631</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Расчёт полинома по заданным значениям</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>36</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Полином</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Результат</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>30</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_8">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>80</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>80</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_9">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>80</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_10">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>431</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_5">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>501</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>631</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Log программы</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1315</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
